### DISCUSSION TITLE
MacOS charset

### DISCUSSION THREAD

**Discussion by Brunoiooo** [espanso/espanso#1900] [Feature requests and Ideas]
Support for all characters :)

**Comment by smeech** [comment]
Would you give us some examples of what's missing?
Are you referring to triggers, or replace, or both?

**Comment by AucaCoyan** [comment]
Hey @Brunoiooo , would you clarify please? 🙏🏼

**Reply by Brunoiooo** [reply]
On macos, char set is bad. Special chars (ąćż...) isn't work(on windows is good display) @AucaCoyan

**Reply by smeech** [reply]
This may be another non-English keyboard problem?

**Reply by Brunoiooo** [reply]
Nope, We had tested on a lot of macos (with polish charset keyboard) and when we had pressed autowrite, we had seen artefacts on polish chars. I was trying changed file charset on  UTF-8 (but this file had set on utf-8). On windows is fine @smeech

**Comment by MasterAlexWest** [comment]
Not sure if is related to this thread but I am unable to use specific Swedish characters å, ä, ö in strings for triggers.

Is this expected or am I missing something in the configuration?

**Comment by smeech** [comment]
Since my original reply I have compiled a long list of Issues possibly relating to non-English keyboards:
https://github.com/espanso/espanso/issues/345
https://github.com/espanso/espanso/issues/498
https://github.com/espanso/espanso/issues/511
https://github.com/espanso/espanso/issues/996
https://github.com/espanso/espanso/issues/1112
https://github.com/espanso/espanso/issues/1133
https://github.com/espanso/espanso/issues/1286
https://github.com/espanso/espanso/issues/1153
https://github.com/espanso/espanso/issues/1609
https://github.com/espanso/espanso/issues/2088

? https://github.com/espanso/espanso/issues/628
? https://github.com/espanso/espanso/issues/680
? https://github.com/espanso/espanso/issues/1028
? https://github.com/espanso/espanso/issues/1327
? https://github.com/espanso/espanso/issues/1347
? https://github.com/espanso/espanso/issues/1408
? https://github.com/espanso/espanso/issues/1439
? https://github.com/espanso/espanso/issues/1507
